taskcluster/kinds/pr/kind.yml

```diff
@@ -0,0 +1,18 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+kind-dependencies:
+    - test
```

Comment:
Worth including `ui` and/or `docker-image`?

Reply:
It should run on PRs if relevant changes are made, I'd have thought?

Reply:
Ah yeah, I think you're right, `2025-01-24 13:02:17,269 - DEBUG - replace_tasks: docker-image-balrog-agent replaced with ILAweleCSDSlOvtke6gtLA by optimization strategy`

it's getting optimized away because nothing relevant changed. I'll add `docker-image` too then

Reply:
```
Exception: In docker-image kind task 'python3.9':
extra keys not allowed @ data['attributes']
```

Or not